src/Plugin/SmsGateway/Twilio.php

```diff
@@ -145,4 +152,64 @@ public function send(SmsMessageInterface $sms_message) {
     return $result;
   }
 
+  /**
```

Comment:
I think this might be outdated review given the API changes in smsframework that expect certain functionality on the plugin.